src/pages/MarkingPage/index.tsx

```diff
@@ -1,27 +1,24 @@
 import { ChevronUpIcon } from '@radix-ui/react-icons'
-import { Box, Card, Flex, Section as RadixUISection, Separator, Text } from '@radix-ui/themes'
-import { instanceToPlain } from 'class-transformer'
-import { keyBy, map, mapValues, sum, sumBy, values } from 'lodash'
+import { Box, Card, Flex, Grid, Section as RadixUISection, Text } from '@radix-ui/themes'
+import { keyBy, mapValues, sumBy, values } from 'lodash'
 import React, { FC, useMemo, useState } from 'react'
 
 import Body from '../../components/pageStructure/Body'
-import Part from '../../components/questionStructure/Part'
-import Question from '../../components/questionStructure/Question'
-import Section from '../../components/questionStructure/Section'
-import { TaskFactory, TaskProps } from '../../components/questionStructure/Task'
 import MarkingToolbar from '../../components/topBars/MarkingToolbar'
+import useActiveIdOnScroll from '../../hooks/interactiveScrollspy'
 import { useAnswers, useMarks, useQuestions, useStudents } from '../../hooks/marking'
 import { Student } from '../../types/marking'
-import MarkInputPanel from './MarkInputPanel'
-import NoAnswerBanner from './NoAnswerBanner'
-import QuestionHeader from './QuestionHeader'
+import MarkableSubmission from './MarkableSubmission'
+import Scrollspy from './Scrollspy'
+import './index.css'
 
 const MarkingPage: FC = () => {
   const { questions, questionsAreLoaded } = useQuestions()
   const { students, studentsAreLoaded } = useStudents()
```

Comment:
Neither `studentsAreLoaded` nor `answersAreLoaded` are used here

Reply:
Yep. Not strickly part of this PRs: they will eventually be used, but I can remove them on main for the meantime (in a dedicated refactor commit).